packages/SwingSet/test/test-terminate.js

```diff
@@ -0,0 +1,18 @@
+import '@agoric/install-ses';
+import path from 'path';
+import { test } from 'tape-promise/tape';
+import { buildVatController, loadSwingsetConfigFile } from '../src/index';
+
+test('terminate', async t => {
+  const configPath = path.resolve(__dirname, 'basedir-terminate/swingset.json');
+  const config = loadSwingsetConfigFile(configPath);
+  const controller = await buildVatController(config);
+  t.equal(controller.bootstrapResult.status(), 'pending');
+  await controller.run();
+  t.equal(controller.bootstrapResult.status(), 'fulfilled');
+  t.deepEqual(controller.bootstrapResult.resolution(), {
```

Comment:
there's a short `capargs` function in some of the other tests to build the serialized Result data, feel free to copy it in here to tidy this up. I keep meaning to move it into `util.js` and share it, but it's only like four lines.

Reply:
In this case I think you mean the `capdata` function, but in any case it would be good to have it in a library.  At this point inserting it would add more code and complexity to already working code than it would eliminate, but if we end up with more test cases (as you have ambitions to happen) I'll add it.